cypher/cypher-docs/src/docs/dev/syntax/expressions.asciidoc

```diff
@@ -70,4 +71,11 @@ String literals can contain the following escape sequences:
 |`\Uxxxxxxxx`|Unicode UTF-32 code point (8 hex digits must follow the `\U`)
 |===================
 
+[[cypher-expressions-number-literals]]
+== Note on number literals
+
+Any number literal may contain an underscore `_` between digits.
+There may be an underscore between the `0x` or `0o` and the digits for hexadecimal and octal literals.
+When using the deprecated way to write an octal (`0`), an underscore is not allowed.
```

Comment:
Isnt the old octal version removed in Neo4j 5 ?

Thus have a note like:

Neo4j version 4.x allows to write an octal with `0`, this was removed in Neo4j 5.

Reply:
true, we started this targeting 4.4, so missed updating this with removals in 5.0